From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Mon, 31 Dec 2018 10:49:48 +0100
Subject: [PATCH] config tuning

allow ovs_... options:
 addon_syntax_check=0

multiple_vlan_aware_bridge_support=1

don't shutdown tap, veth, vmbrvX, fwbr on reload
(as they are not defined in /etc/network/interfaces)

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 etc/network/ifupdown2/ifupdown2.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/network/ifupdown2/ifupdown2.conf b/etc/network/ifupdown2/ifupdown2.conf
index e05c35f..2d2c177 100644
--- a/etc/network/ifupdown2/ifupdown2.conf
+++ b/etc/network/ifupdown2/ifupdown2.conf
@@ -29,7 +29,7 @@ disable_cli_interfacesfile=0
 # are used. But when a mix of scripts and modules are used (which is the
 # default case), you may get false warnings for attributes supported
 # by scripts
-addon_syntax_check=0
+addon_syntax_check=1
 
 # Support executing of ifupdown style scripts.
 # Note that by default python addon modules override scripts with the same
@@ -42,7 +42,7 @@ addon_python_modules_support=1
 # By default ifupdown2 only supports a single vlan filtering bridge
 # on the system. Set this flag to 1 to support multiple vlan
 # filtering bridges
-multiple_vlan_aware_bridge_support=0
+multiple_vlan_aware_bridge_support=1
 
 # ifquery check status strings.
 # By default `ifquery --check` prints the check and
